bin/Cargo.toml

```diff
@@ -29,12 +29,12 @@ libc = "0.2.150"
 log = "0.4.20"
 env_logger = { version = "0.11", default-features = false }
 tokio-util = { version = "0.7.10", features = ["rt", "io"] }
-futures-macro = "0.3.30"
+futures-macro = "0.3.40"
```

Comment:
we should fix whatever script is being used to bump versions

Reply:
its just a sed command, I guess I will have to make it a bit smarter.